ceagle/main.py

```diff
@@ -18,12 +18,12 @@
 
 import flask
 
-from ceagle.blueprints.capacity import capacity
-from ceagle.blueprints.cloud_status import cloud_status
-from ceagle.blueprints.infrastructure import infrastructure
-from ceagle.blueprints.intelligence import intelligence
-from ceagle.blueprints.optimization import optimization
-from ceagle.blueprints.security import security
+from ceagle.services.v1.capacity import capacity
```

Comment:
I would prefer to see different structure: 

from ceagle.v1 import capacity

or if you would like to split api implementation and rest of modules let's make it 

from ceagle.api.v1 import capacity 

so another thing is that it's bad to have directory with module with the same name (we just don't need this)